administrative-sdk/basicAuth.js

```diff
@@ -6,7 +6,6 @@
  max-len,
  no-unused-vars
  */
-
```

Comment:
The removal of newlines shouldn't have been committed, as this type of changes potentially causes merge conflicts.

I'll accept it for now though.